src/main/java/mate/academy/bookstore/controller/BookController.java

```diff
@@ -32,4 +34,15 @@ public BookDto getBookById(@PathVariable Long id) {
     public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
         return bookService.save(requestDto);
     }
+
+    @PutMapping("/{id}")
+    public BookDto updateBookById(@PathVariable Long id,
+                                  @RequestBody CreateBookRequestDto requestDto) {
+        return bookService.update(id, requestDto);
+    }
+
+    @DeleteMapping("/{id}")

```

Comment:
add `no content` response status to here, and `created` for the post endpoint that you already have